marshmallow/schema.py

```diff
@@ -260,6 +258,9 @@ class Meta:
     :param unknown: Whether to exclude, include, or raise an error for unknown
         fields in the data. Use `EXCLUDE`, `INCLUDE` or `RAISE`.
 
+    .. versionchanged:: 3.0.0
```

Comment:
I figured I should mention the change here. I don't think I needed to specify the beta version.